### ISSUE TITLE
Add the ability to build kendo source with gulp

### ISSUE THREAD

**Issue by nenchef** [telerik/kendo-ui-core#3666]
This task includes a revision of the source code distribution and the gulpfile.js in particular. Only several custom tasks are included in the gulpfile.js, which causes the inability to build the entire source with gulp.

**Comment by juancarlosbaezpozos** [comment]
yes please, provide a task to build the entire source !! why just customs parts of it??'
